src/Process/GlobeTileProcessing.js

```diff
@@ -73,9 +73,9 @@ function pointHorizonCulling(pt) {
 
 function horizonCulling(node) {
     const points = node.OBB().pointsWorld;
-
-    for (const point of points) {
-        if (!pointHorizonCulling(point)) {
+    // loop on obb's highest points
```

Comment:
Is `pointsWorld` used by anyone else?

Because if it's only used here, it might be worth only computing the top 4 in `_cPointsWorld()`

Reply:
Same as in #619 then

Reply:
yes, PR updated